src/cbmc/cbmc_solvers.cpp

```diff
@@ -212,8 +216,8 @@ std::unique_ptr<cbmc_solverst::solvert> cbmc_solverst::get_smt2(
 
     if(!*out)
     {
-      error() << "failed to open " << filename << eom;
-      throw 0;
+      throw invalid_user_input_exceptiont(
+        "Failed to open suggested file: " + filename, "--outfile");
```

Comment:
Well, when I work with a deterministic computer program I'm not making suggestions.